src/javascript/Edit/Edit.jsx

```diff
@@ -39,13 +40,13 @@ export const EditCmp = ({
                 sections,
                 values
             },
-            editCallback: nodeUuid => {
+            editCallback: (node, mutateNode) => {
                 if (values[Constants.systemFields.OVERRIDE_SUBMIT_CALLBACK]) {
-                    values[Constants.systemFields.OVERRIDE_SUBMIT_CALLBACK]();
+                    values[Constants.systemFields.OVERRIDE_SUBMIT_CALLBACK](node, mutateNode);
```

Comment:
### ISSUE 2:

We have an issue here:
The language switcher will not perform the renaming operation.
because it's overriding the submit callback to switch the langue, but it should also handle the rename.
Scenario:
- open jcontent
- edit Home page
- rename system name
-switch language: confirmation popup: save changes (no rename handling)
- go back
- jcontent issue, because the rename have not been handled.

So I think we should always do:
envProps.rename(node, mutateNode);
values[Constants.systemFields.OVERRIDE_SUBMIT_CALLBACK](node, mutateNode);

With this, we insure that no matter the override submit callback we always handle the rename on the save.
I thinks it will be even better if we always do it on top of the editCallback, no matter if it's an overrided or not, like: 

> envProps.rename(node, mutateNode);
> if (values[Constants.systemFields.OVERRIDE_SUBMIT_CALLBACK]) {
>____values[Constants.systemFields.OVERRIDE_SUBMIT_CALLBACK](node, mutateNode);
> } else {
> ____const envEditCallback = envEditCallbacks[contentEditorConfigContext.env];
> ____if (envEditCallback) {
> ________envEditCallback(node, mutateNode, contentEditorConfigContext);
> ____}
> }

So we insure the rename is always handled no matter the case of saving.
And we avoid to have to handle it specifically for every kind of save operations
do not forget to remove it from the **envEditCallback**